docs/source/conf.py

```diff
@@ -106,6 +107,7 @@
 
 ## Cutomize the theme
 html_theme_options = {
+    "announcement": "BrainGlobe version 1 is under development! Keep track of the latest developments on <a href='blog/index.html'>the blog page</a>",
```

Comment:
Rephrased this, I thought users won't know what version 1 is, and maybe be a bit confused considering they've been using some tools for nearly 4 years!

```suggestion
    "announcement": "BrainGlobe is undergoing restructuring. Keep track of the latest developments on <a href='blog/index.html'>the blog</a>",
```